components/HomePage/Letter.tsx

```diff
@@ -319,3 +322,24 @@ export const LetterForm = () => {
     </Form>
   );
 };
+
+function getUtmParams() {
+  const utmParams : {[key: string]: string} = {};
+  const sessionUtm = adReferralService.getSessionUtmParams();
+  if (sessionUtm !== null) {
+    if (sessionUtm.source !== null) {
+      utmParams["utm_source"] = sessionUtm.source;
+    }
+    if (sessionUtm.medium !== null) {
+      utmParams["utm_medium"] = sessionUtm.medium;
+    }
+    if (sessionUtm.campaign !== null) {
+      utmParams["utm_campaign"] = sessionUtm.campaign;
+    }
+    if (sessionUtm.content !== null) {
+      utmParams["utm_content"] = sessionUtm.content;
+    }
```

Comment:
🌈 i feel like this would be a lot simpler if we just name the properties w/ the `utm_` prefix within the utm service. any reason for not doing that?